### PR TITLE
[15.0][ADD] Account Receipt Send

### PR DESCRIPTION
**Pull request by Shide**
# Account Receipt Send

- Adds a button to send/print receipts and a new mail template to send it.
- The template is the same as the invoice but replacing "invoice" words by "receipt".

MT-2035 @moduon  @rafaelbn @yajo

### CODE REVIEW

File: account_receipt_send/__manifest__.py [hunk 1]
```diff
@@ -0,0 +1,20 @@
+# Copyright 2023 Eduardo de Miguel - Moduon Team
```
Comment (rafaelbn) [inline account_receipt_send/__manifest__.py]: Change to https://github.com/OCA/account-financial-tools/blob/14.0/account_journal_general_sequence/__manifest__.py#L1

File: account_receipt_send/readme/CONTRIBUTORS.rst [hunk 2]
```diff
@@ -0,0 +1,3 @@
+* `Moduon Team <https://moduon.team>`_:
```
Comment (rafaelbn) [inline account_receipt_send/readme/CONTRIBUTORS.rst]: Change to estandard https://github.com/OCA/account-financial-tools/blob/14.0/account_journal_general_sequence/README.rst#contributors

File: account_receipt_send/README.rst [hunk 3]
```diff
@@ -0,0 +1,84 @@
...
+    :alt: Translate me on Weblate
+.. |badge5| image:: https://img.shields.io/badge/runboat-Try%20me-875A7B.png
+    :target: https://runboat.odoo-community.org/webui/builds.html?repo=OCA/account-invoicing&target_branch=15.0
+    :alt: Try me on Runboat
+
+|badge1| |badge2| |badge3| |badge4| |badge5|
+
+Adds a button to Send and Print Receipts
+Adds a new mail template for receipts
+
```
Comment (rafaelbn) [inline account_receipt_send/README.rst:30]: Even the module is so simple, please add usage section, all modules are made same way so user expect to find exactly same readme even de usage si trivial

https://github.com/OCA/maintainer-tools/tree/master/template/module/readme

- [ ] Add usage section

File: account_receipt_send/README.rst [hunk 4]
```diff
@@ -0,0 +1,84 @@
...
+    :target: https://translation.odoo-community.org/projects/account-invoicing-15-0/account-invoicing-15-0-account_receipt_send
+    :alt: Translate me on Weblate
+.. |badge5| image:: https://img.shields.io/badge/runboat-Try%20me-875A7B.png
+    :target: https://runboat.odoo-community.org/webui/builds.html?repo=OCA/account-invoicing&target_branch=15.0
+    :alt: Try me on Runboat
+
+|badge1| |badge2| |badge3| |badge4| |badge5|
+
+Adds a button to Send and Print Receipts
+Adds a new mail template for receipts
```
Comment (rafaelbn) [inline account_receipt_send/README.rst:29]: - [ ] Add configuration section also mandatory in all modules (products)

https://github.com/OCA/maintainer-tools/blob/master/template/module/readme/CONFIGURE.rst

- Go to Invoicing -> Configuration and activate -> Customer Invoices -> "Sale Receipt"
imagen


This is USAGE:

- Go to Invoicing -> Customers - > Receipts -> Create a new receipt
- Select a customer and a producto and confirm
- Clic in send button (actually doesnt works so there is a BUG)

**Review by rafaelbn — COMMENTED** [review]
Review complete!

Comments done + the button doesn't appears which is the main goal :-)


imagen

**Comment by Shide** [comment]
@rafaelbn Changes done

**Review by rafaelbn — APPROVED** [review]
Great! 👍

**Review by fcvalgar — APPROVED** [review]
LGTM Thank you @Shide

**Review by rafaelbn — APPROVED** [review]
/ocabot merge patch

**Pull request merged**